### PR TITLE
Rspec tests

### PR DESCRIPTION
**Pull request by lekegitrepo**
### Implement tests for all the methods

### CODE REVIEW

**Comment by Ispirett** [comment]
Hey, guys, you need to use the correct git-flow please make a pull request that merges to the development branch.
resubmit for a review

**Comment by Oghenebrume50** [comment]
reviewing...........

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi Team, good job on the test, it works perfectly. :clap:  Let's add a few good practices to the test
Please make sure to have positive and negative test cases for tests. That is, if you test an outcome to be truthy then you should have a test for the false outcome too. 

and fix the merge conflict showing up also. 

Please submit a re-review when the changes are made

File: spec/game_manager_spec.rb [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+      board = init_board
+      board[1] = 'X'
+      expect(game_manager.space_filled?(1)).to eq(true)
+    end
+
+    context 'Current space = empty'
+    it 'Checks if a board position has X or O' do
+      board = init_board
+      board[2] = '-'
+      expect(game_manager.space_filled?(2)).to eq(false)
```
Comment (samgaco) [inline spec/game_manager_spec.rb:41]: :+1:

File: spec/game_manager_spec.rb [hunk 2]
```diff
@@ -35,11 +35,19 @@
...
+			board = init_board
+			board[3] = 'O'
+			expect(game_manager.valid_move?(3)).to eq(false)
+		end
+
+		context 'Valid Move (Board position empty)'
+		it 'Checks if option selected is between 1-9, returns true if empty' do
+			board = init_board
+			board[3] = 'O'
+			expect(game_manager.valid_move?(1)).to eq(true)
```
Comment (samgaco) [inline spec/game_manager_spec.rb]: :white_check_mark:

File: spec/game_manager_spec.rb [hunk 3]
```diff
@@ -67,17 +76,40 @@
       board = init_board
       board[0] = 'X'
       expect(game_manager.current_player).to eq('O')
+    end
+
+		it 'Displays current player symbol based on turn_count, X if even, O if odd' do
+      board = init_board
+      board[0] = 'X'
+			board[1] = 'O'
+      expect(game_manager.current_player).to eq('X')
```
Comment (samgaco) [inline spec/game_manager_spec.rb:84]: :white_check_mark:

File: spec/game_manager_spec.rb [hunk 4]
```diff
@@ -67,17 +76,40 @@
...
+      board[0] = 'X'
+      board[1] = 'O'
+      board[2] = 'X'
+      board[3] = 'O'
+			board[4] = 'X'
+			board[5] = 'O'
+			board[6] = 'X'
+      board[7] = 'O'
+			board[8] = 'X'
+      expect(game_manager.full?).to eq(true)
```
Comment (samgaco) [inline spec/game_manager_spec.rb:111]: :white_check_mark:

**Review by samgaco — APPROVED** [review]
Hi @lekegitrepo and @noomdalv :high_brightness: 

I see all the previous requirements implemented, the project is approved. You may merge to the master branch and submit the project completion form.

----------------
Please, remembet :arrow_down: 

* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

-----------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**